application/apps/indexer/processor/src/grabber.rs

```diff
@@ -620,3 +591,19 @@ fn subtract_what_is_possible(n: u64, m: u64) -> u64 {
         0
     }
 }
+
+/*
+    Should returns positions of all bytes, where new entity ends. It's called
+    "inlet" - point of entry's end.
+    TODO: probably it should be switched to BEGIN, but not END
+*/
+fn get_inlets_plaint_text(buffer: &[u8]) -> Result<(u64, usize), GrabError> {
+    let nl_count = bytecount::count(&buffer, b'\n') as u64;
+    if nl_count == 0 {
+        Ok((0, 0))
+    } else if let Some(offset) = buffer.iter().rposition(|&byte| (&byte == &b'\n' || &byte == &b'\r')) {
```

Comment:
we should either check for `\n` in all places or `\n` and `\r`

Reply:
Probably right way would be:
- check for `\n\r` or `\r\n` and use is as a single line-breaker -> exit
- check for `\n` -> exit
- check for `\r` -> exit
What do you think?